bin/flutter_application_generator.dart

```diff
@@ -0,0 +1,18 @@
+import 'dart:io';
```

Comment:
I'm not sure a folder named bin should be in source control?

Reply:
Yes it's the entry point for a Dart CLI application.

Reply:
oh ok